core/services/workflows/syncer/engine_registry_test.go

```diff
@@ -52,10 +52,6 @@ func TestEngineRegistry(t *testing.T) {
 	// get all
 	es := er.GetAll()
 	require.Len(t, es, 2)
-	require.Equal(t, srv, es[0].Service)
```

Comment:
Why are these incorrect?

Reply:
There are multiple items in the engine and it doesn't return items in insert order

Reply:
We already check the items during other method calls, so checking each item every time seems a bit excessive and feels unnecessary, so I'm removing